src/Stratis.Bitcoin/P2P/SelfEndpointTracker.cs

```diff
@@ -1,5 +1,13 @@
-﻿using System.Net;
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
 using ConcurrentCollections;
+using NBitcoin;
+using Stratis.Bitcoin.Configuration;
+using Stratis.Bitcoin.Configuration.Settings;
+using Stratis.Bitcoin.Connection;
+using Stratis.Bitcoin.Utilities.Extensions;
 
```

Comment:
using Stratis.Bitcoin.Configuration;
using Stratis.Bitcoin.Configuration.Settings;
using Stratis.Bitcoin.Utilities.Extensions;

Directives are not used in this class.

Reply:
yes